src/main/java/com/automattic/elasticsearch/statsd/StatsdService.java

```diff
@@ -205,12 +203,7 @@ public void run() {
                         }
                     }
 
-
-                    try {
-                        Thread.sleep(StatsdService.this.statsdRefreshInternal.millis());
-                    } catch (InterruptedException e1) {
-                        continue;
-                    }
+                    Thread.sleep(StatsdService.this.statsdRefreshInternal.millis());
```

Comment:
I'm not sure if it's still an issue but ES 1.x used to sometimes interrupt here which would kill the event loop without the `try/catch` guard. I think it's safer to leave it.